### PR TITLE
update formatting in extract_filepaths_for_granule README input JSON …

### PR DESCRIPTION
**Pull request by clappn**
## Summary of Changes

Please write a sentence or two summarizing the proposed change.

Updated README for the extract_filepaths_for_granule task because the input JSON was causing manual runs of the lambda to fail.

## Changes

* Added correct JSON keys to allow the input code block to run on a manual Lambda run

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Added JSON lines and tested by running the lambda manually with the JSON

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] User documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] Manual tests (outlined above)
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

**Review by rizbihassan — APPROVED** [review]

File: tasks/extract_filepaths_for_granule/README.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ resource "aws_lambda_function" "extract_filepaths_for_granule" {
 The lambda handler event excepts a dictionary having a list of granules as input. Check the input schema [here](https://github.com/nasa/cumulus-orca/blob/master/tasks/extract_filepaths_for_granule/schemas/input.json) and the configuration schema [here](https://github.com/nasa/cumulus-orca/blob/master/tasks/extract_filepaths_for_granule/schemas/config.json). An example input to the Lambda function can be seen below.
 ```
 {
+      "payload":{
```
Comment (AndrewDorn) [inline tasks/extract_filepaths_for_granule/README.md:83]: Missing indent for this section.

**Review by AndrewDorn — CHANGES_REQUESTED** [review]
Yup, seems correct. The other version seems to be correct for if `os.environ['CUMULUS_MESSAGE_ADAPTER_DISABLED']` is `true`. We've got all sorts of mixed definitions due to the translation levels in that class.

File: tasks/extract_filepaths_for_granule/README.md (hunk 1 above)
Comment (clappn) [reply tasks/extract_filepaths_for_granule/README.md:83]: For clarification, this line needs more or less indentation? I it needs one less tab but wanted to check

Comment (AndrewDorn) [reply tasks/extract_filepaths_for_granule/README.md:83]: I was just noting that `payload` and `granules` were at the same indentation. Yeah, just move everything one tab to the left and it should be fine.

Comment (AndrewDorn) [reply tasks/extract_filepaths_for_granule/README.md:83]: task_config is over-indented as well.

Comment (clappn) [reply tasks/extract_filepaths_for_granule/README.md]: Understood, just pushed the fixes to removed the excess of indentations

**Review by AndrewDorn — APPROVED** [review]

**Pull request merged**